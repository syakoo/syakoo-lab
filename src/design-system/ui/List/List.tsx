import { listStyles } from "./List.css";

type ListProps = {
  children: React.ReactNode;
};

export const List: React.FC<ListProps> = ({ children }) => {
  return <ul className={listStyles.root}>{children}</ul>;
};
