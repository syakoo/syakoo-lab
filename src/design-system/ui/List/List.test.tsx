import { testStories } from "@/test/testStories";

import * as stories from "./List.stories";

describe("List", () => {
  testStories(stories);
});
