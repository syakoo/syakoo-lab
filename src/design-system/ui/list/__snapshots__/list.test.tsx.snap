// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List スナップショット Sample 1`] = `
<body>
  <div>
    <ul
      class="list_listStyles_root__kagjjt0"
    >
      <li
        class="list_listStyles_item__kagjjt1"
      >
        List.Item
      </li>
      <li
        class="list_listStyles_item__kagjjt1"
      >
        List.Item
      </li>
      <li
        class="list_listStyles_item__kagjjt1"
      >
        List.Item
      </li>
    </ul>
  </div>
</body>
`;
